generator-honeycomb/app/templates/.config/_server.js

```diff
@@ -118,5 +118,5 @@ const productionConfig = {
 
 module.exports = Hoek.merge(
   defaultConfig,
-  process.env.NODE_ENV === 'development' ? developmentConfig : productionConfig
+  process.env.NODE_ENV === 'development' ? developmentConfig : productionConfig,
```

Comment:
Trailing commas in function declarations and function calls are valid syntax since ECMAScript 2017